src/main/java/lotto/view/InputView.java

```diff
@@ -12,31 +14,55 @@ private InputView() {
     public static int askPurchaseAmount() {
         System.out.println("구매금액을 입력해주세요.");
         int purchaseAmount = SCANNER.nextInt();
-        validatePurchaseAmount(purchaseAmount);
+        validateGreaterThanZero(purchaseAmount);
         return purchaseAmount;
     }
 
-    private static void validatePurchaseAmount(int purchaseAmount) {
-        if (purchaseAmount <= 0){
+    private static void validateGreaterThanZero(int inputNumber) {
+        if (inputNumber <= 0) {
             throw new IllegalArgumentException("0보다 큰 수만 입력할 수 있습니다.");
         }
     }
 
+    public static int askManualCount() {
+        System.out.println("수동으로 구매할 로또 수를 입력해 주세요.");
+        int manualCount = SCANNER.nextInt();
+        validateGreaterThanZero(manualCount);
+        return manualCount;
+    }
+
+    public static List<String[]> askManualNumbers(int count) {
+        ArrayList<String[]> result = new ArrayList<>();
```

Comment:
<img width="157" alt="image" src="https://user-images.githubusercontent.com/68527004/193707690-284511f2-8301-424f-988c-1a09c532cc2d.png">
코드 수정량을 보고 큰 변경사항 없이 마지막 단계를 잘 반영해주셨구나 싶었습니다 😄 

다만 `String[]` 같이 배열을 그대로 사용하는 부분이나 
```suggestion
        List<String[]> result = new ArrayList<>();
```
요런 부분들은 한번쯤 정리가 되면 좋을 것 같습니다 😄